### PR TITLE
DOCS-26 - Metrics DPM - GA

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request (PR) adds a release note and publishes the following beta article for GA release:
https://help.sumologic.com/docs/metrics/metrics-dpm/

### Ticket

- [Jira - DOCS-26](https://sumologic.atlassian.net/browse/DOCS-26)
- [Asana](https://app.asana.com/0/1202988996948021/1203812552454108/f)

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Comment by jpipkin1** [comment]
@kimsauce or @JV0812, could you review this PR? The release is still a week away, but it would be nice to have the PR ready.

File: blog-service/2023-10-27-metrics.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  - dpm
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+We're excited to announce the release of Metrics DPM, a new screen that provides detailed metrics data points per minute (DPM) information.
+
+Metrics DPM answers the question “What things are sending metrics data, and how much are they sending?” Metrics DPM not only provides high level information about the metrics volume (DPM), but also breaks it down by individual metric names, as well as specific dimensions like cluster, host, and source. With this information, you can determine the items that contribute most to your credit consumption.
```
Comment (kimsauce) [inline blog-service/2023-10-27-metrics.md:16]:
```suggestion
Metrics DPM answers the question, “What things are sending metrics data, and how much are they sending?”. Metrics DPM not only provides high level information about the metrics volume (DPM), you'll also see a breakdown by individual metric name, as well as specific dimensions like cluster, host, and source. With this information, you can determine the items that contribute most to your credit consumption.
```

**Review by kimsauce — APPROVED** [review]
Minor suggestion but otherwise looks good

**Comment by jpipkin1** [comment]
Thanks for the quick review, @kimsauce!

**Pull request merged**